tex/gregoriotex.lua

```diff
@@ -1160,48 +1108,37 @@ local function compile_gabc(gabc_file, gtex_file, glog_file, allow_deprecated)
         .."See the documentation of Gregorio or your TeX\n"
         .."distribution to automatize it.",
         cmd, tex.formatname, tex.jobname)
-  elseif res ~= 0 then
+  else
+    -- Copy the contents of glog_file into warnings.
+    -- Open it for writing so that the LuaTeX recorder knows that gregorio wrote to it.
     local glog = io.open(glog_file, 'a+')
     if glog == nil then
```

Comment:
I'm unsure why we had to downgrade this to a warning for the gtex file, but not for the glog file.